starter/Makefile

```diff
@@ -12,7 +12,7 @@ BUILDFLAGS := -gcflags "all=-N -l" $(BUILDFLAGS)
 endif
 
 # The details of the application:
-binary:=cli
+binary:=startercli
```

Comment:
Lets put somethig very unique as name and meaningfull.

startervarname placeholdercli etc.

StarterCli does generate false positives

Reply:
Resolved via commit: https://github.com/aerogear/charmil/pull/243/commits/d9daf62f3f2be1b9de2af2fc23a9a0814258e776